### DISCUSSION TITLE
Upgrading to 56.0: Does recreating the airbyte_workspace volume on each container result in a loss of configuration data?

### DISCUSSION THREAD

**Discussion by womackn** [airbytehq/airbyte#37414] [Connector Questions]
I'm using Airbyte 40.14 and looking to upgrade to 56.0. The release notes have this indication:

"A breaking change to Docker deployments. Containers no longer run as the root user. If you are upgrading from an older version, the airbyte_workspace docker volume must be recreated to fix the ownership of staging directories under /tmp (in the container)."

The worker, server and airbyte-cron containers use this volume. 

Will recreating the volumes with _docker volume rm airbyte_workspace_ and _docker volume create airbyte_workspace_ inside each container _after_ upgrading result in the loss of any configuration or connector data?

**Comment by marcraminv** [comment]
@womackn I think this volume is used to save the log information.

[a comment by an automated account is not shown]
